.github/ISSUE_TEMPLATE/configure-xx-story-for-the-dashboard.md

```diff
@@ -1,14 +1,55 @@
 ---
-name: Configure XX story for the dashboard
-about: High-level steps for configuring a story for the dashboard
-title: Configure XX story for the dashboard
-labels: 'stories'
+name: Content Request
+about: Use this template to start the content creation process
+title: [Name of Content] - [Dataset, Discovery, or Both]
+projects: 'VEDA Data & Content Ingestion'
 assignees: ''
 
 ---
 
-- [ ] Read the story
-- [ ] Make sure all datasets required have been published
-- [ ] Configure all datasets in veda-config ([Content documentation](../../docs/CONTENT.md) is a good place to start)
-- [ ] Configure story content in veda-config and check preview (dependency on running the dashboard locally or https://github.com/NASA-IMPACT/veda-config/issues/12)
-- [ ] Preview story with story tellers
+## Instructions
+
+Follow the steps in this ticket to get Datasets and Discoveries published to the VEDA Dashboard. For detailed instructions along each step, use [this document](- Take first stab at comms posts for GFOI).
+
+### Dataset or Discovery
+
+First, identify what type of content you are wanting to publish. There are three main options:
+
+1. New data that does not yet exist in VEDA
+2. New Discoveries that only involve graphics or maps of [data already in VEDA](http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/)
+3. New Discoveries that involve datasets that don't yet exist in VEDA
+
+If your content falls into option 1, fill in Dataset in the Title of this ticket. If it is option 2, fill in Discovery. If option 3, fill in Both. This means you will have to ingest the data before completing the configuration of the discovery.
+
+### Configuration Process
+
+Follow the high-level [instructions](https://almanac.io/handbook/veda-product-development-documentation-I3UwKo/getting-from-ingestion-to-discovery-I1ESDqS4NuHVdOx82peffQqMGDDdTyE4) linked above to ingest data or configure the dashboard. You should have the ability to create pull requests in all the necessary Github repositories. To create a pull request, it is recommended to do so from this issue, so that your pull requests will all be linked with this issue. This makes it easy to track for yourself and the VEDA team. To create a pull request from an issue, follow these instructions.
```

Comment:
This instruction moved to : https://nasa-impact.github.io/veda-docs/contributing/dashboard-configuration/